src/spikeinterface/sorters/external/kilosort4.py

```diff
@@ -1,8 +1,8 @@
 from __future__ import annotations
 
 from pathlib import Path
-import os
 from typing import Union
+from packaging.version import parse
```

Comment:
Where are we using parse? I don't see it in the diff. Maybe the original idea was to treat different versions differently?

Reply:
yep indeed that was the original idea, removing :)